src/api/public/apidocs-new/paths/source_project_name_meta.yaml

```diff
@@ -33,3 +33,50 @@ get:
       $ref: '../components/responses/unknown_project.yaml'
   tags:
     - Sources - Projects
+
+put:
+  summary: Write project meta file
+  description: |
+    Write project's meta file. Create the project if it doesn't exist.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: comment
+      required: false
```

Comment:
```suggestion
```
It the parameter is not required, there is no need to set `required` to `false` and this line could be removed.